src/Widgets/configurePisaSalesQuestionnaireWidgets.ts

```diff
@@ -0,0 +1,25 @@
+import { load, Obj } from 'scrivito'
+import {
+  BackendConnectionData,
+  initPisaSalesQuestionnaireWidgets,
+} from 'scrivito-pisasales-questionnaire-builder'
+import { pisaSalesApiUrl } from '../Data/pisaClient'
+import { jwtPisaSalesApiAuth } from '../Data/jwtPisaSalesApiConfig'
+
+export function configurePisaSalesQuestionnaireWidgets() {
+  initPisaSalesQuestionnaireWidgets({ connection: connection() })
+}
+
+async function connection(): Promise<BackendConnectionData> {
+  if (import.meta.env.FORCE_LOCAL_STORAGE) return { apiUrl: null, token: null }
+
+  // Workaround for issue #11896
+  await load(() => Obj.onAllSites().all().count())
```

Comment:
That seems very odd - is this an extra request to the Web Builder backend that happens on _every_  load in the portal?